.github/workflows/pytest-workflow.yml

```
@@ -2,7 +2,7 @@ name: pytest-workflow
 # This workflow runs the pipeline with the minimal test dataset to check that it completes without any syntax errors
 on:
   pull_request:
-    branches: [dev]
+    branches: [dev, master, sentieon]
```

```suggestion
    branches: [dev]
```
And remember this part